python/llm/example/transformers/transformers_int4/GPU/baichuan/README.md

```diff
@@ -1,11 +1,11 @@
 # Baichuan
-In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Baichuan models on any Intel® Arc™ A-Series Graphics. For illustration purposes, we utilize the [baichuan-inc/Baichuan-13B-Chat](https://huggingface.co/baichuan-inc/Baichuan-13B-Chat) as a reference Baichuan model.
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Baichuan models on any Intel® GPUs. For illustration purposes, we utilize the [baichuan-inc/Baichuan-13B-Chat](https://huggingface.co/baichuan-inc/Baichuan-13B-Chat) as a reference Baichuan model.
```

Comment:
In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Baichuan models on any [Intel GPUs](../README.md).

Reply:
fixed. 

And I guess it is not accurate to say "any Intel GPUs", so I also remove the "any".